### DISCUSSION TITLE
Alleluja, alleluja (after Graduale) on a new line?

### DISCUSSION THREAD

**Discussion by Michaelskapel** [DivinumOfficium/divinum-officium#3992] [Q&A] [answered]
I see several updates in the Latin version in which the Graduale and Alleluja are in a way 'decoupled' in the sense that 'Alleluja, alleluja' now starts on a new line instead of following immediately at the end of the Graduale (and thus often on the same line). Is this the new practice (which is actually something that I prefer)?

**Comment by APMarcello3** [comment, marked as answer]
Yes, please do so, as it reflects the printed texts of the Missale Romanum.
